services/api/src/contact_support.py

```diff
@@ -31,25 +31,21 @@ class ContactSupportResource(Resource):
 
     def __init__(self):
         self.CSM_EMAIL_TO_SEND_FROM = os.environ.get("CSM_EMAIL_TO_SEND_FROM")
-        self.CSM_EMAIL_APP_USERNAME = os.environ.get("CSM_EMAIL_APP_USERNAME")
-        self.CSM_EMAIL_APP_PASSWORD = os.environ.get("CSM_EMAIL_APP_PASSWORD")
         self.CSM_EMAILS_TO_SEND_TO = os.environ.get("CSM_EMAILS_TO_SEND_TO")
         self.CSM_TARGET_SMTP_SERVER_ADDRESS = os.environ.get(
             "CSM_TARGET_SMTP_SERVER_ADDRESS"
         )
         self.CSM_TARGET_SMTP_SERVER_PORT = int(
             os.environ.get("CSM_TARGET_SMTP_SERVER_PORT")
         )
+        self.CSM_TLS_ENABLED = os.environ.get("CSM_TLS_ENABLED")
```

Comment:
Can we actually default this to true even in the code so we don't have breaking changes the moment this is deployed in existing deployments without the addition of the new variables. (like CDOT's) I could foresee someone building the latest and deploying it without realizing new environment variables were added.

Reply:
Good point! This has been addressed.